### PR TITLE
50 update user form with clients feedback

### PR DESCRIPTION
**Pull request by baharbazargan62**
## Description

add 'Transgender' and 'Other' options to the gender dropdown list
qualification field should have the following options: 'Cannot read or write', '8th', '10th', '12th', 'Graduate', 'Postgraduate', 'Other'
case status should have the following options: 'Ongoing', 'Acquitted', 'Convicted'
pin code and email fields should be non compulsory: pin code should also be changed in the database and NOT have 'not null' requirement
## Related 
#50 

Fixes #1

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have carefully reviewed my own code
- [ ] I have commented my code
- [ ] I have updated any documentation

### CODE REVIEW

**Comment by Miladepour** [comment]
I'm receiving error 400 (Bad Request) after submitting the form, are you done with these pull requests?

**Comment by baharbazargan62** [comment]
> I'm receiving error 400 (Bad Request) after submitting the form, are you done with these pull requests?

thank you Milad for checking this it has done know please check again

**Comment by natyfromwonderland** [comment]
@Miladepour , are you still getting an error?

**Comment by Miladepour** [comment]
> @Miladepour , are you still getting an error?

Yes, what about you?

**Comment by natyfromwonderland** [comment]
> 

can you see what the error is? Is the pin code in your local database set to be NOT not null?

File: client/src/components/UserForm.js [hunk 1]
```diff
@@ -131,6 +131,8 @@ export default function UserForm() {
             <option value="">Select...</option>
             <option value="male">Male</option>
             <option value="Female">Female</option>
+            <option value="Transgender'">Transgender</option>
```
Comment (Miladepour) [inline client/src/components/UserForm.js:134]: **<option value="Transgender**'**">** in this line there is one single quote extra which sends the wrong data to the database, 
Please change this.
"Transgender"

**Review by Miladepour — CHANGES_REQUESTED** [review]

**Comment by Miladepour** [comment]
> > 
> 
> can you see what the error is? Is the pin code in your local database set to be NOT not null?

Thanks, I fixed that on my database.

**Review by Miladepour — APPROVED** [review]
Thanks, Bahar.
I finished my review and tested it.

**Review by natyfromwonderland — APPROVED** [review]
Thanks Bahar! Could you please inform your team to update their local databases (pin code field shouldn't be not null) and there might be another pull request enhancement to check for phone and email format required for sending info functions.

**Pull request merged**